storage/onpremise/mongodb/variables.tf

```diff
@@ -105,3 +105,33 @@ variable "validity_period_hours" {
   type        = string
   default     = "8760" # 1 year
 }
+
+variable "path" {
+  description = "Path for mounting secrets"
+  type        = string
+  default     = "/mongodb"
+}
+
+variable "adapter_class_name" {
+  description = "Name of the adapter's class"
+  type        = string
+  default     = "ArmoniK.Adapters.MongoDB.TableStorage"
+}
+
+variable "tls" {
+  description = "Whether to deploy MongoDB with TLS"
+  type        = bool
+  default     = true
+}
+
+variable "direct_connection" {
+  description = "Whether we have DirectConnection or not"
+  type        = bool
+  default     = false
+}
```

Comment:
What is direct connection ? Connect to MongoDB explicitly on the primary node only ?

Reply:
This forces the operations to run on the host specified in the connection URI. It won't be compatible with the sharded version as it doesn't work with the srv connection format

Reply:
The sharded version won't be exposed with a srv connection format as the routers with which database users must communicate are exposed with a classic Kube Service. It's the non-sharded version that is exposed with a srv connection format.

That being said I don't see how it could run elsewhere than on the host in the connection URI... Maybe you should explicit this to me IRL